test/extensions/upstreams/http/metadata/BUILD

```diff
@@ -0,0 +1,21 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_test",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+envoy_cc_test(
```

Comment:
envoy_extension_cc_test

Reply:
I added a TODO for this; it requires adding to `extensions_build_config.bzl` which requires a config registration class